server/event.go

```diff
@@ -102,8 +122,8 @@ func (a *ApplicationHandler) ReplayAppEvent(w http.ResponseWriter, r *http.Reque
 // @Failure 400,401,500 {object} util.ServerResponse{data=Stub}
 // @Security ApiKeyAuth
 // @Router /api/v1/projects/{projectID}/events/{eventID} [get]
-func (a *ApplicationHandler) GetAppEvent(w http.ResponseWriter, r *http.Request) {
-	_ = render.Render(w, r, util.NewServerResponse("App event fetched successfully",
+func (a *ApplicationHandler) GetEndpointEvent(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func (a *ApplicationHandler) GetEndpointEvents(w http.ResponseWriter, r *http.Request) {
```

Reply:
Hmm, but we're fetching a single event here based on the `eventID`

Reply:
Oh, disregard then